internal/local/environment.go

```diff
@@ -0,0 +1,56 @@
+package local
+
+import (
+	"fmt"
+	"io/ioutil"
+	"path/filepath"
+	"strings"
+
+	"github.com/10gen/realm-cli/internal/cli/user"
+)
+
+// Environment contains the name, full filepath, and api keys of a profile
+type Environment struct {
+	Name        string
+	Filepath    string
+	Credentials user.Credentials
+}
+
+// LoadProfileEnvs returns a list of each profile's environment containing name, filepath, and api keys
+func LoadProfileEnvs() ([]Environment, error) {
+	dir, dirErr := user.HomeDir()
+	if dirErr != nil {
+		return nil, fmt.Errorf("failed to get CLI profiles: %w", dirErr)
```

Comment:
this returned `err` is ultimately formatted with `"profiles list failed: %w"`, so the wrapped error here is not entirely necessary (it ends up sounding a bit redundant)

same comment goes for below